packages/vats/tools/fake-bridge.js

```diff
@@ -1,7 +1,108 @@
 import { Fail } from '@agoric/assert';
+import { makeTracer, VBankAccount } from '@agoric/internal';
 import assert from 'node:assert/strict';
+import { E } from '@endo/far';
 
-/** @import {ScopedBridgeManager} from '../src/types.js'; */
+const trace = makeTracer('FakeBridge', false);
+
+/**
+ * @import {BridgeHandler, ScopedBridgeManager} from '../src/types.js';
+ * @import {Remote} from '@agoric/vow';
+ */
+
+/** @typedef {{ [address: string]: { [denom: string]: bigint } }} Balances */
+
+export const FAUCET_ADDRESS = 'faucet';
+const INFINITE_AMOUNT = '99999999999';
+
+/**
+ * You can withdraw from the `faucet` address infinitely because its balance is
+ * always huge. When you withdraw, it's as if it is topped up again by a Cosmos
+ * transaction outside the Agoric VM. (Similarly for deposits.)
```

Comment:
It's not immediately obvious how to call the faucet. Consider a helper function or comment that explains `E(fakeBridgeMgr).getBankForAddress(FAUCET_ADDRESS)`

Reply:
I had in mind to wrap this with a "bankManager" helper but deferred. Prompted by your suggestion: [da1314a](https://github.com/Agoric/agoric-sdk/pull/9402/commits/da1314a2e032d0975c3466202c5b0a9b31c7e933)

Reply:
/giphy pour it up